webextension/background.js

```diff
@@ -6,11 +6,15 @@
 
 /* globals browser */
 
+const gURLThankYou = "https://mozilla.github.io/webcompat-blipz-experiment/thanks.html";
```

Comment:
This is also going to change, hopefully we'll have the final URI in the next 2 days.